cnf-certification-test/suite_test.go

```diff
@@ -264,3 +289,102 @@ func incorporateVersions(claimData *claim.Claim) {
 		ClaimFormat:  ClaimFormatVersion,
 	}
 }
+func StartServer() {
+	server := &http.Server{
+		Addr:         ":8084",           // Server address
+		ReadTimeout:  10 * time.Second,  // Maximum duration for reading the entire request
+		WriteTimeout: 10 * time.Second,  // Maximum duration for writing the entire response
+		IdleTimeout:  120 * time.Second, // Maximum idle duration before closing the connection
+	}
+	HandlereqFunc()
+
+	http.HandleFunc("/runFunction", RunHandler)
+
+	fmt.Println("Server is running on :8084...")
+	if err := server.ListenAndServe(); err != nil {
+		panic(err)
+	}
+}
+
+// Define an HTTP handler that triggers Ginkgo tests
+func RunHandler(w http.ResponseWriter, r *http.Request) {
+	Buf = bytes.NewBufferString(aString)
+	log.SetOutput(Buf)
```

Comment:
Logs are not complete, they show only the part written by logrus, not the part written by Ginkgo. This is because ginkgo defines the log output in an init function and locks it down by calling an internal function to initialize it. I could not find a way so far to redirect ginkgo's output without changing ginkgo. Since we are planning to move our of Ginkgo, we can wait until the move

Reply:
yah i also tried to do that with alot of things didnt help